File: DAM_1er_año/Programacion/Python/Resolucion_Examen_Final/ej1.py
def suma_recursiva(n):
    if n == 0:
        return 0
    elif n == 1:
        return 1
    else:
        return n + suma_recursiva(n-1)


n1 = int(input("Dime un número: "))
numero = suma_recursiva(n1)

print(f'La suma de los primeros {n1} números es: {numero}')

File: DAM_2º-año_1º/Programación/UD5-Ej Ampliacion/src/Ej1.java
import java.util.Arrays;
import java.util.Scanner;

/**
 * Desarrolla un programa que pida al usuario introducir una lista de números
 * separados por comas, el programa deberá ignorar, si los hubiera, los espacios
 * entre números y comas. También deberá ignorar datos no numéricos, si los
 * hubiera. El programa deberá mostrar los números ordenados de mayor a
 * menor.
 **/
public class Ej1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Dime una lista de números separados por comas: ");
        String numero = scanner.nextLine().trim();

        String[] numeros = numero.replaceAll(" ", "").split(",");


    }
}
